generators/entity-client/templates/angular/src/main/webapp/app/entities/_entity-management-dialog.component.html

```diff
@@ -100,7 +100,7 @@ <h4 class="modal-title" id="my<%= entityClass %>Label" jhiTranslate="<%= keyPref
             </div>
             <%_ } else if (['Instant', 'ZonedDateTime'].includes(fieldType)) { _%>
             <div class="d-flex">
-                <input id="field_<%= fieldName %>" type="datetime-local" class="form-control" name="<%= fieldName %>" [(ngModel)]="<%= entityInstance %>.<%= fieldName %>"
+                <input id="field_<%= fieldName %>" type="datetime-local" class="form-control" name="<%= fieldName %>" [(ngModel)]="<%= fieldName %>"
```

Comment:
Why?

Reply:
Because it's a String property on the ts component deduced from the model object attribute typed with 'Moment'

Reply:
ok

Reply:
Sorry, but I'm not sure to understand.
 - In order to be editable in an input, the attribute type should be a string
 - In order to be handled well, the model element attribute should be typed as moment

* I tried piping ngModel binding and event, but it do not work (the field is emptied when filled).

That said, I didn't found another solution than having a string component attribute deduced from the model element (interface?) one while using model element's setter, then converting it back to the model element's attribute while saving

Reply:
Hmm, ok, can you implement all remaining comments. We can see what can be done for this after that. for now let it be.

Also please add unit tests for any new file added here or in ng-jhipster